cluster-operator/src/main/java/io/strimzi/operator/cluster/model/TopicOperator.java

```diff
@@ -53,8 +48,11 @@
     private static final String NAME_SUFFIX = "-topic-operator";
     private static final String CERTS_SUFFIX = NAME_SUFFIX + "-certs";
     protected static final String TLS_SIDECAR_NAME = "tls-sidecar";
-    protected static final String TLS_SIDECAR_VOLUME_NAME = "tls-sidecar-certs";
-    protected static final String TLS_SIDECAR_VOLUME_MOUNT = "/etc/tls-sidecar/certs/";
+    protected static final String TLS_SIDECAR1_VOLUME_NAME = "eo-certs";
+    protected static final String TLS_SIDECAR1_VOLUME_MOUNT = "/etc/tls-sidecar/eo-certs/";
+    protected static final String TLS_SIDECAR2_VOLUME_NAME = "cluster-ca-certs";
+    protected static final String TLS_SIDECAR2_VOLUME_MOUNT = "/etc/tls-sidecar/cluster-ca-certs/";
```

Comment:
a nit ... the number 1 and 2 just after `SIDECAR` let me think that we are defining volume names and mounts for 2 sidecars while we are defining two volume names and mounts for one sidecar. Imho, the number should be right after `VOLUME` (i.e. `TLS_SIDECAR_VOLUME1_NAME` and so on) or using meaningful names (i.e. `TLS_SIDECAR_CLUSTER_CA_VOLUME_` bla bla bla)